.github/workflows/zap_export.yml

```diff
@@ -116,11 +116,14 @@ jobs:
             -v $(pwd)/.output:/.output\
             nycplanning/library:ubuntu-01d9e5eddf65dbdea3eb0c500314963b5ec6246a library archive -f /templates/${{ matrix.entity }}.yml -v $VERSION -s -l -o csv --compress
 
-      - name: Export open data to edm-publishing
+      - name: Export to edm-private and edm-publishing
         env:
           VERSION: ${{ steps.version.outputs.version }}
         run: |
+          ./zap.sh upload_crm_do ${{ matrix.entity }} $VERSION
+
           if ${{ matrix.open }};
           then
-          ./zap.sh upload_do ${{ matrix.entity }} $VERSION
+          ./zap.sh upload_internal_do ${{ matrix.entity }} $VERSION
```

Comment:
Should this step be happening for non-open datasets? Since this is the internal upload function

Reply:
for non-open datasets, the CRM upload function and Data Library archiving above this section is actually all we need

"internal" and "visible" exports only relate to the open datasets

Reply:
So just to clarify one more time - "non-open" datasets only go to recipes through data library, while the others go to publishing and private (private being the "internal" one)

Reply:
exactly